Source/SvgDocument.cs

```diff
@@ -444,7 +444,7 @@ private static T Create<T>(XmlReader reader) where T : SvgDocument, new()
             if (styles.Any())
             {
                 var cssTotal = string.Join(Environment.NewLine, styles.Select(s => s.Content).ToArray());
-                var stylesheetParser = new StylesheetParser(true, true);
+                var stylesheetParser = new StylesheetParser(true, true, tolerateInvalidValues: true);

```

Comment:
TolarateInvalidValues is needed so that Invalid Property Values are still read. Could be Too that ExCss is not correctly annotated yet.

This fixed a Bug: But now we have to set tolarateInvalidValues to still get all values
https://github.com/TylerBrinks/ExCSS/commit/cea80ccf5c392e2bc5cd0e853c94167c811d315e